zeppelin-interpreter/src/main/java/org/apache/zeppelin/interpreter/InterpreterGroup.java

```diff
@@ -161,4 +161,17 @@ public boolean equals(Object o) {
   public int hashCode() {
     return id != null ? id.hashCode() : 0;
   }
+
+  public void close() {
+    for (List<Interpreter> session : sessions.values()) {
+      for (Interpreter interpreter : session) {
+        try {
+          interpreter.close();
+        } catch (InterpreterException e) {
+          LOGGER.warn("Fail to close interpreter: " + interpreter.getClassName());
```

Comment:
would the exception stack be useful?
just to LOGGER.warn( .... , e);?

Reply:
Fixed